src/PrestaShopBundle/Resources/config/services/adapter/group.yml

```diff
@@ -17,3 +17,15 @@ services:
     class: 'PrestaShop\PrestaShop\Adapter\Feature\GroupFeature'
     arguments:
       - '@prestashop.adapter.legacy.configuration'
+
+  PrestaShop\PrestaShop\Adapter\Customer\Group\CommandHandler\AddCustomerGroupHandler:
```

Comment:
add autowire: true, public: false and remove repository argument.
```

Reply:
True, and the same for the other handler (which will also need the repository injected)

Reply:
Done